server/src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/DateHistogramAggregationBuilder.java

```diff
@@ -501,4 +502,14 @@ public boolean equals(Object obj) {
     public Version getMinimalSupportedVersion() {
         return Version.V_EMPTY;
     }
+
+    @Override
+    protected void validateSequentiallyOrdered(String type, String name, Consumer<String> addValidationError) {}
+
+    @Override
+    protected void validateSequentiallyOrderedWithoutGaps(String type, String name, Consumer<String> addValidationError) {
+        if (minDocCount != 0) {
+            addValidationError.accept("parent histogram of " + type + " aggregation [" + name + "] must have min_doc_count of 0");
```

Comment:
Do we need to think about extended/hard bounds here?  I'm pretty sure the answer is "no", but I want to say it out loud.

Reply:
We never have in the past. I think it's ok because those'll just make the result wider without adding any gaps.